packages/web-runtime/src/App.vue

```diff
@@ -70,18 +70,24 @@ export default defineComponent({
     ...mapGetters(['configuration', 'capabilities', 'getSettingsValue']),
     ...mapGetters('runtime/auth', ['isUserContextReady', 'isPublicLinkContextReady']),
     layout() {
-      if (!this.$route.name || !isAuthenticationRequired(this.$router, this.$route)) {
-        return LayoutPlain
+      const pageLayouts = {
```

Comment:
```suggestion
      const pageLayouts: Record<PageLayout, VueComponentSomething> = { 
```

.... again satisfies would be very nice here - then the object could simply satisfy `Record<PageLayout, any>` without narrowing or widening the type